plugins/outputs/azure_data_explorer/README.md

```diff
@@ -21,7 +22,7 @@ This plugin writes metrics collected by any of the input plugins of Telegraf to
   # database = ""
```

Comment:
Minor difference, but the hashes don't match in the README.md and the SampleConfig. In the readme endpoint_url and database have a hash but in the code it is missing. Was this intentional?

Reply:
Yes, endpoint_url and database are required, rest all are optional.